plugins/mcreator-core/blockly/mcreator_extensions.js

```diff
@@ -203,3 +203,147 @@ Blockly.Extensions.register('dimension_custom_list_provider',
         this.appendDummyInput().appendField(new Blockly.FieldDropdown(
             arrayToBlocklyDropDownArray(javabridge.getListOf("dimension_custom"))), 'dimension');
     });
+
+Blockly.Extensions.register('searchable_entity_provider',

```

Comment:
I think you should apply those changes directly in the current extensions. With this new dialog, the old one is, in my opinion, useless and bad. Did you have a specific reason to keep old extensions?

Reply:
Not really, I wasn't sure if it would need an element converter so I kept them separated at first